components/mixins/experience-event/experienceevent-claim-process.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-claim-process",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Claim Process",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": [
+    "financial_services",
+    "health_and_life_sciences",
+    "high_tech"
+  ],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Claim details such as claim policy ID and claim creation steps.",
+  "definitions": {
+    "claim-process": {
+      "properties": {
+        "xdm:disputes": {
+          "type": "object",
+          "properties": {
+            "xdm:claimProcess": {
+              "type": "object",
+              "properties": {
+                "xdm:claimID": {
+                  "title": "Claim ID",
+                  "type": "string",
+                  "description": "ID of the claim being submitted."
+                },
+                "xdm:claimStartDate": {
+                  "title": "Claim Start Date",
+                  "type": "string",
+                  "format": "date-time",
+                  "description": "Date the claim was successfully submitted."
+                },
+                "xdm:claimStatus": {
+                  "title": "Claim Status",
+                  "type": "string",
+                  "enum": ["initiated", "in progress", "approved", "declined"],
+                  "description": "Captures the status of the claim for any given point in the application process."
+                },
+                "xdm:claimSteps": {
+                  "title": "Claim Steps",
+                  "$ref": "https://ns.adobe.com/xdm/datatypes/tool-usage",
+                  "description": "Extension of toolUsage data type."
+                },
+                "xdm:claimType": {
+                  "title": "Claim Type",
+                  "type": "string",
+                  "description": "Type of claim submitted."
+                },
+                "xdm:policy": {
```

Comment:
Should we name this as claimPolicy as "policy" is too generic to be under "xdm" namespace.